thirdweb/modules/nft.py

```diff
@@ -12,28 +12,42 @@
 
 
 class NftModule(BaseModule):
-    """ 
+    """
     NFT Methods
     """
 
     address: str
     __abi_module: NFT
 
+
     def __init__(self, address: str, client: Web3):
+        """
+        Initializing the class attributes
+        """
         super().__init__()
         self.address = address
+        """
+        NFT module contract address [https://thirdweb.com/: Dashboard: Project ➝ NFT Module]
+        """
+
         self.__abi_module = NFT(client, address)
+        """
+        The ABI makes calls the EVM. Client is default 'Web3' and Address is nft module contract address.
+        """
 
     def mint(self, arg: MintArg) -> NftType:
         """
-        Mints a new token
+        - Mints a new token. 
+        - Arguments passed: Note, a class is used -> MintArg(name, description, image_uri, properties) *Preferrably, using a link
+        - Returns the 'NftMetadata(name,description,image,properties,id,uri)
         """
         return self.mint_to(self.get_signer_address(), arg)
+        
 
     def mint_to(
-            self,
-            to_address: str,
-            arg: MintArg,
+        self,
+        to_address: str,
+        arg: MintArg,
     ) -> NftType:
         """
         Mints a new token
```

Comment:
we should probably maintain consistency here by displaying a format similar to what is written for the [mint method](https://github.com/nftlabs/nftlabs-sdk-python/pull/15/files#diff-85ff59513ad491407f1f7bcaad29cf1d428e525190145629436e39407893033dL27) or mentioning that the usage is same as the `mint` method.